### PR TITLE
docs: initial documentation

### PR DESCRIPTION
**Pull request by ashitakah**
v1.0 doc

- Script for autogen doc
- Distribute the contracts in core and relays
- Clean external folder and get only the desired contracts
- All contracts have specific docu and 1 how to.

I would like to create more tutorials for the migration. Also I would like to have a css style.

Merge with the refactor is needed

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The MIT License (MIT)
```
Comment (0xGorilla) [inline LICENSE:1]: Please use instead AGPL 3.0, copy it from here: https://github.com/defi-wonderland/sidechain-oracles/blob/dev/LICENSE

File: README.md [hunk 2]
```diff
@@ -1,33 +1,49 @@
-<div align="center"><strong>Keep3r Framework</strong></div>
+# Prophet Core
 
-## Running tests
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/defi-wonderland/prophet-core/blob/main/LICENSE)
```
Comment (0xGorilla) [inline README.md:3]: Also change it here. And you are linking to prophet, watch out

File: solidity/contracts/core/AutomationVault.sol [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: MIT
```
Comment (0xGorilla) [inline solidity/contracts/core/AutomationVault.sol:1]: All of these changes in file structure should be in a separate PR

File: docs/src/SUMMARY.md [hunk 4]
```diff
@@ -1,21 +1,34 @@
 # Summary
 
-- [Home](README.md)
+# Documentation
+
+- [Getting Started](content/intro/README.md)
+
+  - [XKeeper](content/intro/xkeeper.md)
+
+- [How to use]
```
Comment (0xArdy) [inline docs/src/SUMMARY.md:9]:
```suggestion
- How to use
```

**Review by 0xArdy — CHANGES_REQUESTED** [review]
After this fix it should deploy a new preview

File: README.md [hunk 5]
```diff
@@ -1,33 +1,49 @@
...
+## Contributors
 
-The scripts and documentation in this project are released under the [MIT License](LICENSE)
+Keep3r Framework was built with ❤️ by [Wonderland](https://defi.sucks).
 
-# Contributors
+Wonderland is a team of top Web3 researchers, developers, and operators who believe that the future needs to be open-source, permissionless, and decentralized.
 
-Maintained with love by [Wonderland](https://defi.sucks). Made possible by viewers like you.
+[DeFi sucks](https://defi.sucks), but Wonderland is here to make it better.
```
Comment (0xGorilla) [inline README.md:49]:
```
## Contributors

Keep3r Framework was built with ❤️ by [Wonderland](https://defi.sucks).

Wonderland is the largest core development group in web3. Our commitment is to a financial future that's open, decentralized, and accessible to all. 

[DeFi sucks](https://defi.sucks), but Wonderland is here to make it better.
```

File: LICENSE (hunk 1 above)
Comment (ashitakah) [reply LICENSE:1]: 👍

File: solidity/contracts/core/AutomationVault.sol (hunk 3 above)
Comment (ashitakah) [reply solidity/contracts/core/AutomationVault.sol:1]: 👍

File: README.md (hunk 2 above)
Comment (ashitakah) [reply README.md:3]: 👍

File: README.md (hunk 5 above)
Comment (ashitakah) [reply README.md:49]: 👍

File: docs/src/SUMMARY.md (hunk 4 above)
Comment (ashitakah) [reply docs/src/SUMMARY.md:9]: 👍

File: docs/src/content/howto/step_by_step.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+### Step 4: Configuration of jobs and selectors
+
+1. **Approval of jobs:**
+
+   - To enable task executions, you need to approve specific jobs that the Automation Vault will interact with and will be allowed.
+
+   - Additionally, you need to approve specific function selectors for each approved job. This ensures that only designated functions within the approved jobs can be executed.
+
+2. **Approval of selectors:**
```
Comment (0xJabberwock) [inline docs/src/content/howto/step_by_step.md:31]: The sub-section title is displayed after its corresponding content.

File: docs/src/content/intro/README.md
Comment (0xJabberwock) [inline docs/src/content/intro/README.md:1]: The content of this file makes multiple references to `Keep3r Framework`, including a link to the old repo. The license statement should be properly updated as well.

File: docs/src/content/relays/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+- The [`Keep3rBondedRelay`](./keep3r_bonded_relay.md): Introduces dynamic bonding requirements for automation vaults, enhancing security by validating tasks with bonded keepers.
+
+- The [`GelatoRelay`](./gelato_relay.md): Acts as a centralized hub for managing and executing tasks from the Gelato network, streamlining automation processes.
+
+- The [`OpenRelay`](./open_relay.md): Manages and facilitates the execution of tasks from various bots, ensuring efficient automation processes and fee payments.
+
+For more detailed information about each contract, please refer to the respective documentation pages.
+
+For more technical details about the interfaces of the core contracts, please refer to the `Interfaces` section in the technical documentation.
```
Comment (0xJabberwock) [inline docs/src/content/relays/README.md:17]: Interfaces of core contracts are mentioned here, but the content should be about relays' interfaces.

File: docs/src/content/relays/keep3r_relay.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# Keep3r Relay
+
+The `Keep3rRelay` is meticulously crafted to orchestrate and streamline task executions from the Keep3r network, offering a cohesive solution for efficient automation processes and fee payments.
+
+## Key Features
+
+- **Task Coordination**: The contract seamlessly coordinates task executions with an `AutomationVault`, ensuring a harmonious and streamlined automation process.
+
+- **Keeper Authorization**: Validation mechanisms guarantee that only authorized Keepers within the Keep3r network can initiate and execute tasks, enhancing security and control.
```
Comment (0xJabberwock) [inline docs/src/content/relays/keep3r_relay.md:9]: In this case, `Keepers` word should be written uncapitalized, as it has been made other times.

File: docs/src/content/relays/open_relay.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+- **Event Transparency**: Upon successful execution, the contract emits the `AutomationVaultExecuted` event, providing transparent insights into executed tasks and relay activities.
+
+## Gas Metrics
+
+- `GAS_BONUS`: 53,000
+- `GAS_MULTIPLIER`: 12,000
+- `BASE`: 10,000
+
+These gas metrics play a vital role in optimizing gas costs during task execution.
```
Comment (0xJabberwock) [inline docs/src/content/relays/open_relay.md:21]: Do the gas metrics optimize gas costs? 🤨

File: docs/src/content/core/automation_vault_factory.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+# Automation Vault Factory
+
+The `AutomationVaultFactory` contract is specifically designed to handle the deployment and management of AutomationVaults within the on-chain automation ecosystem. As a factory, it plays a crucial role in streamlining the creation of new automation vaults, each serving as a dedicated entity for job execution, relay management, and payment processing.
+
+## Key Features
+
+- **Efficient Deployment**: The primary function of the factory is to efficiently deploy new instances of AutomationVaults. This allows users to manage automation tasks, associated relays and handle balances.
+
+- **Automation Vaults Listing**: Enables users to retrieve a list of deployed automation vaults, allowing for efficient tracking and monitoring.
```
Comment (0xJabberwock) [inline docs/src/content/core/automation_vault_factory.md:9]: This file displays twice `AutomationVaults`. I propose normalizing it to `automation vaults`, so it is more cohesive with the rest of the docs.

**Review by 0xJabberwock — CHANGES_REQUESTED** [review]
My copy editing report can be found below. 🧐

File: README.md [hunk 11]
```diff
@@ -1,33 +1,51 @@
-<div align="center"><strong>Keep3r Framework</strong></div>
+# Prophet Core
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply README.md:1]: HAHAH 👍

File: README.md [hunk 12]
```diff
@@ -1,33 +1,51 @@
...
-```
+⚠️ The code has not been audited yet, tread with caution.
 
-In order to just run unit tests, run:
+## Overview
 
-```bash
-yarn test:unit
-```
+XKeeper is a public good for on-chain automation. It should be very easy to create a job which can be run both by Keep3r, Gelato, Autonolas, or any other keeper out there. We have also deployed compatible relays to ensure seamless integration, but Keep3r Frameworks is designed to work with any keeper of your choice, providing you with the flexibility to select the one that best suits your needs.
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply README.md:9]: 👍

File: README.md [hunk 13]
```diff
@@ -1,33 +1,51 @@
...
-In order to run unit tests and run way more fuzzing than usual (5x), run:
+## Setup
 
-```bash
-yarn test:unit:deep
+This project uses [Foundry](https://book.getfoundry.sh/). To build it locally, run:
+
+```sh
+git clone git@github.com:defi-wonderland/prophet-core.git
+cd keep3r-framework-core
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply README.md:17]: 👍

File: README.md [hunk 14]
```diff
@@ -1,33 +1,51 @@
...
+yarn build
 ```
 
-In order to just run e2e tests, run:
+### Available Commands
 
-```bash
-yarn test:e2e
-```
+Make sure to set `MAINNET_RPC` environment variable before running end-to-end tests.
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply README.md:24]: 👍

File: README.md [hunk 15]
```diff
@@ -1,33 +1,51 @@
...
+XKeeper was built with ❤️ by [Wonderland](https://defi.sucks).
 
-The scripts and documentation in this project are released under the [MIT License](LICENSE)
+Wonderland is the largest core development group in web3. Our commitment is to a financial future that's open, decentralized, and accessible to all.
 
-# Contributors
+[DeFi sucks](https://defi.sucks), but Wonderland is here to make it better.
 
-Maintained with love by [Wonderland](https://defi.sucks). Made possible by viewers like you.
+[DeFi sucks](https://defi.sucks), but Wonderland is here to make it better.
```
[the remark this answers is not in the document]
Comment (ashitakah) [reply README.md:51]: 👍

File: docs/src/content/relays/README.md (hunk 7 above)
Comment (ashitakah) [reply docs/src/content/relays/README.md:17]: 👍

File: docs/src/content/relays/keep3r_relay.md (hunk 8 above)
Comment (ashitakah) [reply docs/src/content/relays/keep3r_relay.md:9]: 👍

File: docs/src/content/relays/open_relay.md (hunk 9 above)
Comment (ashitakah) [reply docs/src/content/relays/open_relay.md:21]: you right

File: docs/src/content/howto/step_by_step.md (hunk 6 above)
Comment (ashitakah) [reply docs/src/content/howto/step_by_step.md:31]: 👍

File: docs/src/content/core/automation_vault_factory.md (hunk 10 above)
Comment (ashitakah) [reply docs/src/content/core/automation_vault_factory.md:9]: I like it, nice one

**Pull request merged**